server/locale/FR/mutations/gplq.js

```diff
@@ -76,5 +76,47 @@ export default {
             })
         ],
         ...textLogoMutations
+    ],
+    'layout:flex': [
+        [
+            'default',
+            {
+                logo: Logo.PP_PAYPAL.WHITE,
+                headline: [
+                    {
+                        tag: 'xsmall'
+                    },
+                    {
+                        tag: 'medium'
+                    }
+                ],
+                subHeadline: [{ tag: 'small', br: ['paiement en 4X.'] }],
+                disclaimer: ['default']
+            }
+        ],
+        [
+            'ratio:1x1',
+            {
+                subHeadline: [{ tag: 'small', br: ['achats'] }],
+                styles: [
+                    `.message__sub-headline { font-size: 1.2rem; } .message__disclaimer { bottom: 12%; }`,
+                    `@media screen and (max-width: 240px) { .message__sub-headline { font-size: 0.9rem; } }`
```

Comment:
Similar to the ratio:1x1 comment for the gpl mutation file where I think this can be removed since it's shared across both of the mutation files. For more specific GPLQ styles for 1x1, I found that adding ```@media (min-width: 140px) {.message__headline {font-size: 9.5vw;} .message__sub-headline {font-size: 6vw;}``` in the 1x1 styles array makes the sizing a bit better.